src/main/java/io/stargate/sgv2/jsonapi/exception/mappers/ThrowableToErrorMapper.java

```diff
@@ -88,17 +89,21 @@ private static CommandResult.Error handleDriverException(
       return handleAllNodesFailedException((AllNodesFailedException) throwable, message);
     } else if (throwable instanceof ClosedConnectionException) {
       return ErrorCode.SERVER_CLOSED_CONNECTION
-          .toApiException(message)
-          .getCommandResultError(Response.Status.INTERNAL_SERVER_ERROR);
+          .toApiException("(DriverException/ClosedConnectionException) %s", message)
+          .getCommandResultError(Response.Status.GATEWAY_TIMEOUT);
```

Comment:
Why here is `GATEWAY_TIMEOUT` not `BAD_GATEWAY`?

Reply:
Let me double-check; you are right seems odd.

Reply:
@Hazel-Datastax yes, you are right: should be BAD_GATEWAY -- changing. Thank you!